Automattic/jetpack
# VideoPress: add isExtensionEnabled() helper

Fixes #

#### Changes proposed in this Pull Request:

* VideoPress: add isExtensionEnabled() helper


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






Since the `isEstensionEnabled()` is not used, yet, you may want to test
* check the TS implementation

<img width="782" alt="image" src="https://user-images.githubusercontent.com/77539/198696539-e5d4d666-9112-4ee6-8294-bc13a5e1636c.png">

* the app doesn't break
* and check the code visually
